BUILD.bazel

```diff
@@ -47,9 +47,6 @@ clang_format_check(
         "./**/vendor/**",
         # Rust cargo build dirs
         "./**/target/**",
-        # Meson build dirs
-        "./build-out/**",
-        "./build-bin/**",
```

Comment:
Are we still copying stuff to `build-bin` or `build-out` during any of the build flows?  If yes, we should leave this in for now.

Reply:
Ah yes, good catch. Added back.